### DISCUSSION TITLE
Не включается автоматический обмен, только ручная выгрузка\загрузка

### DISCUSSION THREAD

**Discussion by InPleasure** [Tseka85/opencart3-exchange1c#33] [Q&A]
Здравствуйте
Прошу подсказать что не правильно настроил: Не включается автоматический обмен, только ручная выгрузка\загрузка
Модуль последней версии Version 1.6.4.8 (неофициальная)

Подробно в картинках
1. https://musefy.ru/1/vklyucheno.jpg - модуль включен !
2. https://musefy.ru/1/vyklyucheno.jpg - в панели показывает что выключен (((
3. https://musefy.ru/1/Ne-Soxraniaetsia.jpg - при ручной выгрузке не сохраняется файл - выгрузка заказа в xml происходит
4. https://musefy.ru/1/System.jpg - настройки системы
5. https://musefy.ru/1/OC3.0.3.7.jpg

Или где можно детально почитать. Заранее СпасибО!

**Comment by Tseka85** [comment]
Добрый день , к сожалению эта сборка пока не тестировалась на OC 3.0.3.7

**Reply by InPleasure** [reply]
> Добрый день , к сожалению эта сборка пока не тестировалась на OC 3.0.3.7

Дмитрий, могу я Вам выложить готовую локально развернутую версию в виде архива wamp64.rar для настройки\тестирования на OC 3.0.3.7? Могу ссылку в личку )
